server/src/main/java/org/apache/druid/server/coordinator/helper/DruidCoordinatorBalancer.java

```diff
@@ -105,14 +105,17 @@ private void balanceTier(
     }
 
     Map<Boolean, List<ServerHolder>> partitions = servers.stream()
-                                                      .collect(Collectors.partitioningBy(ServerHolder::isMaintenance));
+                                                      .collect(Collectors.partitioningBy(ServerHolder::isInMaintenance));
```

Comment:
`.collect()` is not aligned with `.stream()`. Suggested to break the whole expression:
```java
Map<Boolean, List<ServerHolder>> partitions =
    servers.stream().collect(Collectors.partitioningBy(ServerHolder::isInMaintenance));
```

Reply:
✅